docs/guides/vnc_selenium.rst

```diff
@@ -149,6 +149,16 @@ An example of the yaml is below:
            desired_capabilities:
                platform: LINUX
                browserName: 'chrome'
+               # for the selenium-server version 3.4.0 you will need to use 
+               # following capabilities if you have written 'firefox' in browserName
```

Comment:
default suggestion for browserName is 'chrome' above, so these should be commented out for default config right?